.github/workflows/test-e2e.yml

```diff
@@ -6,21 +6,22 @@ on:
       - master
 jobs:
   run-static-checks:
-    uses: ./.github/workflows/staticcheck.yaml
-  run-e2e-test-ios:
+   uses: ./.github/workflows/staticcheck.yaml
+  build-detox-app:
     needs: run-static-checks
     runs-on: macos-latest
```

Comment:
What about trying a large runner (Intel or even M1)? What do you think?

Reply:
Since we can't use M1 runner for e2e for a detox/jest version issue, I just changed the runner with a large intel one at the moment